pkgs/top-level/python-packages.nix

```diff
@@ -7611,6 +7613,8 @@ in {
 
   spacy = callPackage ../development/python-modules/spacy { };
 
+  spacy_legacy = callPackage ../development/python-modules/spacy/legacy.nix { };
```

Comment:
Please normalize the attribute name. According to the [python contributing guidelines](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#contributing-guidelines), package names within nix should follow [PEP 0503](https://www.python.org/dev/peps/pep-0503/#normalized-names)

`spacy-legacy`